query-graphs-js/src/graphPath.ts

```diff
@@ -1575,6 +1600,19 @@ function advancePathWithDirectTransition<V extends Vertex>(
       continue;
     }
 
+    if (edge.overrideCondition) {
+      const isOverridden = overrideConditions.get(edge.overrideCondition.label) === true;
```

Comment:
So this code technically deals with empty `Map`s by assuming that no label means `false`. Which sounds fine at first glance, but other code that checks for satisfiability doesn't behave like this (and will instead always return `false` for override condition satisfiability if the `Map` is empty).

It would be better to ensure all labels are in `Map` at the start of `buildQueryPlan()` as mentioned in the previous comment.

Reply:
Agreed. As it is this is un-updated remains from the `Set` approach.
[2391f43](https://github.com/apollographql/federation/pull/2902/commits/2391f43bfcb9e698f38ee7da77dee9696076c82c)